exporter/containerimage/writer.go

```diff
@@ -46,6 +46,8 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const attestationManifestArtifactType = "application/vnd.docker.attestation.manifest.v1+json"
```

Comment:
This value should probably be added to https://github.com/moby/buildkit/blob/f198c3849aebe9b1b1ddd7286e831834014355ec/docs/attestations/attestation-storage.md too, right? :eyes: